### ISSUE TITLE
Uglify Error - Unexpected token punc «,», expected punc «:»

### ISSUE THREAD

**Issue by jgolebiewski** [500tech/angular-tree-component#240]
I'm using angular-seed with systemjs-builder and during build.prod process it stops on uglifying file angular-tree-component.umd.js.

If you paste code from this file here
https://skalman.github.io/UglifyJS-online/

you can get the same result.

@version angular-tree-component 3.2.4

> Parse error: Unexpected token punc «,», expected punc «:»
>Line 20349, column 14
>20348:     var array = {
>20349:         chunk, compact, concat, difference, differenceBy,
>20350:         differenceWith, drop, dropRight, dropRightWhile, dropWhile,

**Comment by sjdemoor** [comment]
I'm getting a similar error during (karma) unit tests (see below). 

```
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  SyntaxError: Unexpected token ','
  at webpack:///~/angular-tree-component/dist/angular-tree-component.umd.js:20348:0 <- src/tests.entry.ts:153619
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  SyntaxError: Unexpected token ','
  at webpack:///~/angular-tree-component/dist/angular-tree-component.umd.js:20348:0 <- src/tests.entry.ts:153619

PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 0 of 0 ERROR (1.097 secs / 0 secs)
```

**Comment by chendachao** [comment]
+1

**Comment by adamkleingit** [comment]
Hi @jgolebiewski , @sjdemoor , @chendachao ,

Can you try with the new version 3.3.0?
BTW - in 3.3.0 the UMD bundle is already uglified.

Although I find this strange, it should be an ES5 file to begin with.

**Comment by jgolebiewski** [comment]
Hi,
Now building process is Ok but durnig launch I'm getting this error

> Module not already loaded loading "lodash/defaultsDeep"
> node_modules/angular-tree-component/dist/angular-tree-component.umd.js:2:27961

**Comment by adamkleingit** [comment]
@jgolebiewski - error or warning?

**Comment by jgolebiewski** [comment]
Error

**Comment by myieye** [comment]
I now get the same error. Just ran npm update/install and now I can't start that application.

**Comment by adamkleingit** [comment]
@myieye @jgolebiewski Can you try removing the node_modules folder and reinstalling?

**Comment by Nicolas1701** [comment]
I get the same error since the update 3.3.1

_Error: Module not already loaded loading "lodash/defaultsDeep"
node_modules/angular-tree-component/dist/angular-tree-component.umd.js:2:27961_

I can't start my application anymore.
Hope someone can provide a solution...

**Comment by jb201** [comment]
Also experiencing the same issue 

Error: Module not already loaded loading "lodash/defaultsDeep"

**Comment by ahmedsivac911** [comment]
Using SystemJS and angular-seed, having the same issue, been stuck on it for two days. Really hope someone provides any kind of solution.

**Comment by gpapp** [comment]
See my comment in issue #262 for a workaround of the lodash/* issues.

**Comment by ahmedsivac911** [comment]
We're using angular-seed, and there is no system.config.js file, but i tried adding variations of your lines from the comment in project-config, no luck...

**Comment by adamkleingit** [comment]
Should be fixed in 3.4.1
